src/main/scala/slickless/HListShape.scala

```diff
@@ -7,7 +7,7 @@ import slick.ast.MappedScalaType
 import slick.lifted.{ Shape, ShapeLevel, FlatShapeLevel, MappedProductShape, MappedProjection }
 
 final class HListShape[L <: ShapeLevel, M <: HList, U <: HList : ClassTag, P <: HList]
-    (val shapes: Seq[Shape[_, _, _, _]]) extends MappedProductShape[L, HList, M, U, P] {
+    (val shapes: Seq[Shape[_<: ShapeLevel, _, _, _]]) extends MappedProductShape[L, HList, M, U, P] {
```

Comment:
It's related to slick version bump (to 3.2.0), the first commit.
The cross compilation seems transparent